src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNet/AspNetMvcIntegration.cs

```diff
@@ -90,22 +93,52 @@ public static Scope CreateScope(ControllerContextStruct controllerContext)
                     }
                 }
 
+                string routeUrl = route?.Url;
+                string areaName = (routeValues?.GetValueOrDefault("area") as string)?.ToLowerInvariant();
+                string controllerName = (routeValues?.GetValueOrDefault("controller") as string)?.ToLowerInvariant();
+                string actionName = (routeValues?.GetValueOrDefault("action") as string)?.ToLowerInvariant();
```

Comment:
Is the `GetValueOrDefault` a method of `routeValues` or is using an extension method with Sigil?

Reply:
`GetValueOrDefault()` is defined in the library here: https://github.com/DataDog/dd-trace-dotnet/blob/e51c8d2cfd396df40d363d14e61fe93cee3b0f14/src/Datadog.Trace/ExtensionMethods/DictionaryExtensions.cs#L10

Is that a problem? The method will only be used in newer versions of the tracer so I assume it's fine, or is this another case where mismatched auto/manual versions will cause issues?

Reply:
Is a normal dictionary, no problem at all, I haven't use that extension method that's why I asked 🙂 Thanks!